src/client/pac/PacTypes.ts

```diff
@@ -75,3 +66,5 @@ export type ActiveOrgOutput = {
     UserId : string,
     EnvironmentId: string,
 }
+
+export type PacActiveOrgListOutput = PacOutputWithResultList<ActiveOrgOutput>;
```

Comment:
The Result object output here from `pac org who` is a single object, not a collection.